packages/x-date-pickers/src/internals-fields/index.ts

```diff
@@ -0,0 +1,15 @@
+// We don't add those exports to the regular `@mui/x-date-pickers/internals`,
+// Because they rely on date-fns, which is not used by all applications.
+export {
+  useField,
```

Comment:
I'd say that we should still look into removing the direct dependency of `date-fns` from `useField.utls` going forward 🤔

Reply:
Yes I think we all agree on that
I requires new methods in the adapter so it was not feasable for a simple POC
But the components can't be stable with hardcoded date-fns stuff